js_modules/dagster-ui/packages/ui-core/src/assets/AssetNodeOverview.tsx

```diff
@@ -209,6 +212,20 @@ export const AssetNodeOverview = ({
           )}
         </Box>
       </AttributeAndValue>
+      <AttributeAndValue label="Owners">
+        {assetNode.owners &&
+          assetNode.owners.map((owner, idx) =>
+            owner.__typename === 'UserAssetOwner' ? (
+              <UserAssetOwnerWrapper key={idx}>
+                <UserDisplay key={idx} email={owner.email} size="very-small" />
+              </UserAssetOwnerWrapper>
+            ) : (
+              <Tag icon="people" key={idx}>
+                {owner.team}
+              </Tag>
+            ),
```

Comment:
This should probably be wrapped in a `Box` with `gap: 4` set rather than relying on the browser's default spacing between the list element